torch/csrc/jit/python/pybind_utils.cpp

```diff
@@ -63,12 +66,27 @@ IValue toIValue(py::handle obj, const TypePtr& type, c10::optional<int32_t> N) {
           scalar = at::Scalar(THPUtils_unpackComplexDouble(obj.ptr()));
         } else if (THPUtils_checkDouble(obj.ptr())) {
           scalar = at::Scalar(THPUtils_unpackDouble(obj.ptr()));
+        } else if (torch::is_symint_node(py::handle(obj))) {
+          save_symint = true;
+          scalar = at::Scalar(7777777);
```

Comment:
Do we want to #define this constant to ensure it remains consistent from all the files?

Reply:
The constant value doesn't matter, I just tried to make it something distinctive so that you'd notice if you were accidentally using it. If you like, we could also just zero initialize it, or max_int initialize it. Consistency unnecessary